### PR TITLE
feat: add app icon

### PR DESCRIPTION
**Pull request by guilhermedeandrade**
This PR will add the app icon for both Android and iOS.

The Android icons were generated using [Android Asset Studio](https://github.com/romannurik/AndroidAssetStudio), while the iOS ones [App Icon Generator](https://appicon.co/).

### CODE REVIEW

File: ios/reactNativeWithMoxy/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 1]
```diff
@@ -1,53 +1,62 @@
 {
   "images" : [
     {
+      "filename" : "40.png",
```
Comment (acostalima) [inline ios/reactNativeWithMoxy/Images.xcassets/AppIcon.appiconset/Contents.json:4]: I'm now wondering whether or not we should have a convention when it comes to name these assets. If we need to work with a designer, it should be crystal clear to both developers and designers where the icon fits through its name.

**Example:** if the app icons for iPhone are stored in MOXY's cloud in the directory `projectX/app-icons/ios/iphone`, each app icon could be named after something like `20x20@2x.png`.

I'm not exactly sure what should be the approach here. Shall we discuss this topic with our designers?

Comment (guilhermedeandrade) [reply ios/reactNativeWithMoxy/Images.xcassets/AppIcon.appiconset/Contents.json:4]: Yes, I think we should. It would be nice to have this standardized.

Comment (guilhermedeandrade) [reply ios/reactNativeWithMoxy/Images.xcassets/AppIcon.appiconset/Contents.json]: Done!

File: ios/reactNativeWithMoxy/Images.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
     {
-      "filename" : "180.png",
+      "filename" : "180x180@3x.png",
       "idiom" : "iphone",
       "scale" : "3x",
       "size" : "60x60"
     },
     {
-      "filename" : "1024.png",
+      "filename" : "1024x1024.png",
```
Comment (acostalima) [inline ios/reactNativeWithMoxy/Images.xcassets/AppIcon.appiconset/Contents.json:52]: For consistency's sake:

```suggestion
      "filename" : "1024x1024@x1.png",
```
Don't forget to rename the file. 😉

Comment (guilhermedeandrade) [reply ios/reactNativeWithMoxy/Images.xcassets/AppIcon.appiconset/Contents.json:52]: Sure 👍

**Comment by guilhermedeandrade** [comment]
> Missing app icons for iPad 🙏

Added!

**Pull request merged**